docs/add-ons/crossplane.md

```diff
@@ -39,26 +39,40 @@ You can optionally customize the Helm chart that deploys `Crossplane` via the fo
 ```
 
 ### Crossplane AWS Provider Deployment
-AWS Provider for Crossplane gets deployed by default when you enable `enable_crossplane = true`.
-The below configuration helps you to upgrade the AWS provider version and lets you define custom IAM policies to manage AWS resources through IRSA.
+This module provides options to deploy the following AWS providers for Crossplane. These providers disabled by default, and it can be enabled using the config below.
 
-Crossplane requires Admin like permissions to create and update resources similar to Terraform deploy role.
+ - [AWS Provider](https://github.com/crossplane/provider-aws)
+ - [Provider Jet AWS](https://github.com/crossplane-contrib/provider-jet-aws)
```

Comment:
Terrajet AWS Provider is the name in the repo?

Reply:
It's not the name of the repo but they mentioned this(`Terrajet AWS Provider`) in README. I will refer this everywhere in the docs

Reply:
done